src/components/cards/CardItem.vue

```diff
@@ -11,8 +11,7 @@
 			:tabindex="0"
 			class="card"
 			@click="openCard"
-			@keyup.self="handleCardKeyboardShortcut"
-			@mouseenter="focus(card.id)">
+			@keyup.self="handleCardKeyboardShortcut">
```

Comment:
As discussed i think we should still keep the focus logic but instead ignore the mouseenter if the shortcutlock is set https://github.com/nextcloud/deck/blob/main/src/components/cards/CardItem.vue#L211